ge/org.osate.ge.ba/src/org/osate/ge/ba/ui/properties/EditEmbeddedTextDialog.java

```diff
@@ -51,34 +53,51 @@
 import org.eclipse.ui.handlers.IHandlerService;
 import org.eclipse.xtext.resource.XtextResource;
 import org.eclipse.xtext.resource.impl.ListBasedDiagnosticConsumer;
+import org.eclipse.xtext.xbase.lib.Pair;
 import org.osate.ba.aadlba.BehaviorTransition;
 import org.osate.ge.swt.SwtUtil;
 
 public class EditEmbeddedTextDialog extends MessageDialog {
-	public static String WIDGET_ID_TEXT = "org.osate.ge.ba.behaviortransition.editdialog.text";
-	public static String WIDGET_ID_CONFIRM = "org.osate.ge.ba.behaviortransition.editdialog.confirmation";
+	private static String WIDGET_ID = "org.osate.ge.ba.behaviortransition.editdialog";
+	public static String WIDGET_ID_TEXT = WIDGET_ID + ".text";
+	public static String WIDGET_ID_CONFIRM = WIDGET_ID + ".confirmation";
 	private final EmbeddedXtextAdapter xtextAdapter;
 	private final ExtendedModifyListener textValidator;
 	private final IHandlerService service;
+	private final int styledTextStyle;
+	private final GridData styledTextLayoutData;
 	private IHandlerActivation undoHandler;
 	private IHandlerActivation redoHandler;
 	private StyledText styledText;
-	private String replacementText;
-
+	private String modifiedSrc;
+	private Pair<String, String> modifiedSrcToEnteredValue;
```

Comment:
Remove. Doesn't seem to serve a purpose.

Reply:
Refer to other comments about why this exists.  We can talk about better ways to do this though.  I return the value from the styled text when editing XtextResources and the entire source for modifying XtextDocuments.